src/templates/product.js

```diff
@@ -1,20 +1,119 @@
 import React from "react"
 import { graphql } from "gatsby"
+import {
+  SingularProductContainer,
+  HeroSection,
+  ImageContainer,
+  MainImage,
+  SmallImageContainer,
+  SmallImage,
+  TextContainer,
+  ProductType,
+  ProductName,
+  ProductDescription,
+  IconSection,
+  IconSectionHeading,
+  IconContainer,
+  Icon,
+  IconImage,
+  IconCaption,
+  FeatureSpecSection,
+  FeatureIcon,
+  FeatureSpecHeading,
+  FeatureSpecFeatures,
+  Feature,
+  FeatureName,
+  FeatureImage,
+  FeatureDescription,
+} from "../components/singular-product/singular-product-styles"
 
+import Layout from "../components/layout"
 export default function Product({ data }) {
   const products = data.allPrismicProduct.edges[0].node
 
-  return <h1>{products.data.product_title}</h1>
+  return (
+    <Layout>
+    <SingularProductContainer>
+      <HeroSection>
+        <ImageContainer>
+          <MainImage></MainImage>
+          <SmallImageContainer>
+            <SmallImage></SmallImage>
+            <SmallImage></SmallImage>
+            <SmallImage></SmallImage>
+            <SmallImage></SmallImage>
+            <SmallImage></SmallImage>
+          </SmallImageContainer>
+        </ImageContainer>
+        <TextContainer>
+          <ProductType>{products.data.product_type}</ProductType>
```

Comment:
Are these changes from the other PR? https://github.com/Scout-NU/powerhouse-site/pull/48#pullrequestreview-638265192 

If so, can you make sure you don't have stuff from other PRs in another in the future?